basculechecks/capabilitiesvalidator_test.go

```diff
@@ -307,14 +316,20 @@ func TestGetCapabilities(t *testing.T) {
 		t.Run(tc.description, func(t *testing.T) {
 			assert := assert.New(t)
 			m := map[string]interface{}{CapabilityKey: tc.keyValue}
+			if len(tc.key) > 0 {
+				m = map[string]interface{}{tc.key[len(tc.key)-1]: tc.keyValue}
+				for i := len(tc.key) - 2; i >= 0; i-- {
+					m = map[string]interface{}{tc.key[i]: m}
+				}
+			}
```

Comment:
This might be a good candidate for a helper func like `createDummyClaimsMap` just so we understand it just by looking at the name. If folks are interested, they can look at the details in the function. Just my HO though 😅

Reply:
😅 I guess this is a little confusing...